### ISSUE TITLE
Unnecessary Alt-F1/F2 "drive" entries after installing "snap" package manager: "!/dev/loop"

### ISSUE THREAD

**Issue by delafer** [elfmz/far2l#1845]
after installing "snap" package management system I have a lot of strange entries 
when selecting a disk/path for the left-right panel by pressing Alt-F1 or Alt-F2

I have a lot of entries (and even duplicate entries) which looking like:
```
/var/lib/snapd/snap/snapd/20092/                            |   !/dev/loop0
/var/lib/snapd/snap/snapd/20092/                            |   !/dev/loop0
/var/lib/snapd/snap/wine-platform-6-stable/19/     |   !/dev/loop1
/var/lib/snapd/snap/wine-platform-6-stable/19/     |   !/dev/loop1
/var/lib/snapd/snap/wine-platform-runtime/349/   |   !/dev/loop2
/var/lib/snapd/snap/wine-platform-runtime/349/   |   !/dev/loop2
```

**Comment by delafer** [comment]
![image](https://github.com/elfmz/far2l/assets/5951449/18632197-bf58-4f22-9355-ad7b684e4adb)
![image](https://github.com/elfmz/far2l/assets/5951449/96de3880-2a85-4520-b193-ee941a4aff76)

**Comment by delafer** [comment]
it's certainly not a terrible problem, but it's very annoying and just kind of ugly :-( ...... Is there any possibility to filter or remove them?

**Comment by atolismesh** [comment]
Simply adjust your settings- press AltF1 -> F9 -> Exceptions.
Detailed in #1458

**Comment by delafer** [comment]
It's works! you are the best!!! :+1: 
Kindly please consider to include ;/var/lib/snapd/* into the default exclusions list as well.
Snap / Snapd is pretty popular and other users have same issues...

![image](https://github.com/elfmz/far2l/assets/5951449/b5176056-3dae-4253-be72-7b790d5c51fd)

**Comment by akruphi** [comment]
You can compare your current & default values by `far:config` command (now in far2l it's draft without ability to change value of parameter).

Now parameter **System.DriveExceptions** has only `...;/var/snap/*;...`

**Comment by delafer** [comment]
> You can compare your current & default values by `far:config` command (now in far2l it's draft without ability to change value of parameter).
> 
> Now parameter **System.DriveExceptions** has only `...;/var/snap/*;...`

yeah, and if I'm understanding correctly (please correct me if I'm wrong):
   -  this is not enough so far as I have `...;/var/lib/snapd/*;...` entries..
